tcms/testruns/models.py

```diff
@@ -268,14 +266,24 @@ class TestExecutionStatus(TCMSActionModel):
         'FAIL': '#cc0000'
     }
 
-    complete_status_names = (PASSED, 'ERROR', FAILED, WAIVED)
-    failure_status_names = ('ERROR', FAILED, BLOCKED)
-    passed_status_names = (PASSED, WAIVED)
-    idle_status_names = (IDLE,)
+    from django.utils.translation import gettext_lazy as _
+    POSITIVE = 'P', _('Positive')
+    NEGATIVE = 'N', _('Negative')
+    NEUTRAL = 'NN', _('Neutral')
+
     chart_status_names = ('pass', 'fail', 'idle', 'other')
 
     id = models.AutoField(db_column='case_run_status_id', primary_key=True)
     name = models.CharField(max_length=60, blank=True, unique=True)
+    status = models.CharField(
+        max_length=60,
+        choices=[POSITIVE, NEGATIVE, NEUTRAL]
+    )
```

Comment:
This is weight, not status. Different statuses can have varying weights depending on how important they are.  For example ERROR (-1) can mean a problem with test infra, while BLOCKED (-10) can mean something else, while FAILED (-100) can mean test actually failed. The actual meaning is to be decided by the user so having variability here will give them this option.  A fixed positive, negative & neutral doesn't give them this. It is like hard-coding these 3 statuses.